fashion-example-query/README.md

```diff
@@ -14,6 +14,8 @@
 First of all, read up on [Jina 101](https://github.com/jina-ai/jina/tree/master/docs/chapters/101) so you have a clear understanding of how Jina works. We're going to refer to those concepts a lot. We assume you already have some knowledge of Python and machine learning.
 
 This is an extension of the Hello World example, but here we use QueryLanguage to filter the results by category.
```

Comment:
we use QueryLanguage to create an index per category, thus having results filtered by categories